src/uct/sm/mm/mm_iface.c

```diff
@@ -520,14 +520,16 @@ static UCS_CLASS_INIT_FUNC(uct_mm_iface_t, uct_md_h md, uct_worker_h worker,
                                   sizeof(uct_mm_recv_desc_t),
                                   UCS_SYS_CACHE_LINE_SIZE,
                                   &mm_config->mp,
-                                  mm_config->fifo_size * 2,
+                                  512,
                                   uct_mm_iface_recv_desc_init,
                                   "mm_recv_desc");
     if (status != UCS_OK) {
         ucs_error("Failed to create a receive descriptor memory pool for the MM transport");
         goto err_close_signal_fd;
     }
 
+    ucs_mpool_grow(&self->recv_desc_mp, mm_config->fifo_size * 2);
```

Comment:
why need to make the mpool grow here?

Reply:
so the first allocated chunk would be small, to reduce initial memory usage